pkgs/by-name/au/autosubst2/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  haskellPackages,
+  fetchFromGitHub,
+}:
+
+haskellPackages.mkDerivation {
+  pname = "autosubst2";
+  version = "unstable-2022-07-04";
```

Comment:
Versions should start with a number (→ `0-unstable-2022-07-04`)

Reply:
Hi @pluiedev ! Thanks for the review! I am not sure about this. Is there a concrete standard for this version number for unstable packages? I have seen both versions (`unstable-` and `0-unstable-`) of the version number in many packages.

Reply:
See pkgs/README.md — the packages that aren't prefixed by 0- are packages that we haven't gotten around renaming yet :p The leading number is necessary for version comparisons AFAICT

Reply:
Thanks! I changed it. I may also need to change the version number of packages maintained by me :D